packages/react-native-reanimated/plugin/.gitignore

```diff
@@ -0,0 +1,2 @@
+lib
```

Comment:
I am not sure if we shouuld add it to top level .gitignore instead.

Reply:
This is plugin's .gitignore file 😅